src/interfaces/current-user-interface.ts

```diff
@@ -1,11 +1,10 @@
 /** The Current User (aka: Keycloak) interface. */
 export interface CurrentUserIF {
   email: string
-  firstName: string
-  fullName: string
+  firstname: string
   keycloakGuid: string
-  lastName: string
+  lastname: string
   loginSource: string
   roles: Array<string>
-  userName: string
+  type: string
 }
```

Comment:
This interface is used by the authentication store, which gets its data from the KC token (not from the auth "users/@me" call).

Sample data:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/1acb0e03-d8ea-40a2-b830-dcc96f87d16b)

Don't use THIS interface it you want to type the userInfo object.